src/NServiceBus.Core.Analyzer.Tests/NServiceBus.Core.Analyzer.Tests.csproj.DotSettings

```diff
@@ -0,0 +1,2 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
+	<s:String x:Key="/Default/CodeInspection/CSharpLanguageProject/LanguageLevel/@EntryValue">CSharp72</s:String></wpf:ResourceDictionary>
```

Comment:
Should we put this into the solution-level file instead of adding a project file?

Reply:
I'm happy with that as well. Seems the build server isn't too happy about this anyway? I wanted to keep the changes minimal.